apps/cookbook/src/app/showcase/slides-showcase/slides-showcase.component.html

```diff
@@ -1,4 +1,10 @@
 <div class="example">
+  <p>
+    <em
+      ><strong>Please note:</strong>The slides do not like beeing switched between mobile- and
```

Comment:
Like using the dev-tools or what do you mean?

Reply:
Well, mobile-view is only accessible from dev-tools - so yea..added a note about it

Reply:
added clarifying comment